src/sets.jl

```diff
@@ -73,3 +73,64 @@ function LinearAlgebra.norm(::AbstractVector{<:AbstractJuMPScalar})
           "`@constraint(model, norm(x) <= t)` should now be written as ",
           "`@constraint(model, [t; x] in SecondOrderCone())`")
 end
+
+"""
+    SOS1
+
+SOS1 (Special Ordered Sets type 1) object than can be used to constrain a 
+vector `x` to a set where at most 1 variable can take a non-zero value, all
+others being at 0. 
+
+The `weights`, when specified, induce an ordering of the variables; as such, 
+they should be unique values. The *k*th element in the set corresponds to the
+*k*th weight in `weights`. See [here](http://lpsolve.sourceforge.net/5.5/SOS.htm)
+for a description of SOS constraints and their potential uses.
+
+This is a shortcut for the `MathOptInterface.SOS1`
```

Comment:
"for the `MathOptInterface.SOS1` set."

All sentences should be full sentences that end in a full stop.